CONTRIBUTING.md

```diff
@@ -12,9 +12,17 @@ Welcome, and thanks in advance for your help!
 
 ## Setup
 
-It is recommended to use Node v14 or v16 for development.
-
-Then, to begin development fork repository and run `npm install` in its root folder.
+Pre-Reqs:
+* Python 3.7
+* [poetry](https://python-poetry.org/docs/) (if you use multiple versions of Python be sure to install it with python 3.7)
+* Perl (used in the tests)
```

Comment:
I don't think Perl is really needed

Reply:
Apologies, just now seeing this. I can remove the perl reference if you want, but it is [required](https://github.com/mklenbw/serverless-python-requirements/blob/7715237f8ac41b5889ea2388affcc1c7f7493d2d/test.js#L53) if we want to run the tests. I actually couldn't get the tests to run without it.

Reply:
oh, that's interesting to learn. Thank you @jameskbride 🙇